web_src/js/features/repo-issue-edit.js

```diff
@@ -78,16 +72,16 @@ async function onEditContent(event) {
             dz.removeAllFiles(true);
             dropzone.querySelector('.files').innerHTML = '';
             for (const el of dropzone.querySelectorAll('.dz-preview')) el.remove();
-            fileUuidDict = {};
             disableRemovedfileEvent = false;
 
             for (const attachment of data) {
-              const imgSrc = `${dropzone.getAttribute('data-link-url')}/${attachment.uuid}`;
               dz.emit('addedfile', attachment);
-              dz.emit('thumbnail', attachment, imgSrc);
+              if (/\.(jpg|jpeg|png|gif|bmp|svg)$/i.test(attachment.name)) {
```

Comment:
Try to get the mime type into here and use it instead.

Reply:
The attachment variable has no type attribute. This information is obtained from the database. The database does not record the file type.